### PR TITLE
Fix defaulting for urls.

### PR DESCRIPTION
**Pull request by MatthewDolan**

### CODE REVIEW

File: src/lightstep-datasource/partials/config.html [hunk 1]
```diff
@@ -70,7 +73,10 @@ <h3 class="page-heading">HTTP</h3>
       <input class="gf-form-input" type="text"
              ng-model='ctrl.current.jsonData.dashboardURL' placeholder="https://app.lightstep.com"
              bs-typeahead="getSuggestUrls"  min-length="0"
-             ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/"></input>
+             ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/"
+             ng-init='if (!ctrl.current.jsonData.dashboardUR) { ctrl.current.url = "https://app.lightstep.com" }'
```
Comment (arya) [inline src/lightstep-datasource/partials/config.html]: is `dashboardUR` a typo?

Comment (arya) [reply src/lightstep-datasource/partials/config.html]: Should this update `dashboardURL` instead of `url`?

Comment (MatthewDolan) [reply src/lightstep-datasource/partials/config.html]: fixed.

File: dist/lightstep-datasource/partials/config.html [hunk 2]
```diff
@@ -70,7 +73,10 @@ <h3 class="page-heading">HTTP</h3>
       <input class="gf-form-input" type="text"
              ng-model='ctrl.current.jsonData.dashboardURL' placeholder="https://app.lightstep.com"
              bs-typeahead="getSuggestUrls"  min-length="0"
-             ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/"></input>
+             ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/"
+             ng-init='if (!ctrl.current.jsonData.dashboardUR) { ctrl.current.url = "https://app.lightstep.com" }'
```
Comment (arya) [inline dist/lightstep-datasource/partials/config.html]: I think you forgot to build because this one still has the typo.

**Pull request merged**